packages/xcoobee-cookie-kit-react/src/CookieKitPopup.js

```diff
@@ -665,20 +679,47 @@ export default class CookieKitPopup extends React.PureComponent {
                 </div>
               </button>
               { isCountrySelectShown && (
-                <div className={`${BLOCK}__country-picker-select`}>
-                  { countryCodes.map(cCode => (
-                    <button
-                      type="button"
-                      key={`country-flag-${cCode}`}
-                      className={`xb-cookie-kit__button ${BLOCK}__country-picker-button`}
-                      onClick={() => this.handleCountryChange(cCode)}
-                      title={cCode}
-                    >
-                      <div className={`${BLOCK}__flag`}>
-                        <ReactCountryFlag code={cCode} svg />
-                      </div>
-                    </button>
-                  ))}
+                <div className={`${BLOCK}__country-picker-container`}>
+                  <button
+                    type="button"
+                    className="xb-cookie-kit__button"
+                    onClick={e => this.handleScroll(e, "up")}
+                  >
+                    <img
+                      src={caretUp}
+                      alt="caret-up"
```

Comment:
If I understand correctly, this button is used in both the popup theme and overlay theme. In the popup theme we scroll up and down. In the overlay theme we scroll left and right. Is that correct? If so, maybe a more generic term would be scroll forwards and scroll backwards.

Reply:
Actually no. We added these buttons only for one theme – for overlay. And now we scroll up and down instead of left and right.